### DISCUSSION TITLE
2023년 05월 04일

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#51] [General]
# 2023년 05월 04일

LOG 몸 상태가 너무 안 좋아서 네오에게 재택한다고 말씀드리고 집에서 재택을 했다. 네오도 어제 병가를 냈다고 하시던데 아마 나와 같지 않으셨을까 싶다...ㅋㅋㅋ 어제 검사해보니 A형 독감이라고 나와서 다른 사람들에게 옮기지 않기 위해서라도 재택을…

http://vero.wiki/2023-05-04/

**Comment by kevstevie** [comment]
완쾌하세요 베로가 없으니 루터가 너무 조용하네요

**Reply by Cyma-s** [reply]
경비 아저씨는 좋아하실지도... 걱정 감사합니다 🥹

**Comment by carsago** [comment]
베로 빨리 돌아와

**Reply by Cyma-s** [reply]
ㅋㅋㅋㅋㅋ 내일 감

**Comment by drunkenhw** [comment]
왜 아픈거 말안하셨죠?

**Reply by Cyma-s** [reply]
그야 안 물어봤으니까?ㅋㅋㅋㅋㅋ
